templates/cloud/hybrid-cloud.html

```diff
@@ -0,0 +1,409 @@
+{% extends "cloud/_base-cloud.html" %}
+
+{% block title %}What is a hybrid cloud?{% endblock %}
+
+{% block meta_description %}Hybrid cloud is a cloud computing architecture that consists of a public cloud, a private cloud and a hybrid cloud manager. It enables organisations to optimise their infrastructure costs by always running their workloads where it makes more sense from an economical standpoint.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/19g24YRYzF5dBpUKI1QzTG7fXKj1z0OvwgFll69dFoOs/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-7">
+			<h1>What is private cloud?</h1>
```

Comment:
Something weird has happened with the indentation again, also: 
```suggestion
			<h1>What is hybrid cloud?</h1>
```